### PR TITLE
[Doc Improvement][Docx format removed][3099832]

### PR DESCRIPTION
**Pull request by v-sdhakshina**

### CODE REVIEW

File: api-reference/beta/api/onlinemeeting-list-transcripts.md [hunk 1]
```diff
@@ -123,9 +112,6 @@ HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
-    "@odata.context": "https://graph.microsoft.com/beta/$metadata#users('ba321e0d-79ee-478d-8e28-85a19507f456')/onlineMeetings('MSo1N2Y5ZGFjYy03MWJmLTQ3NDMtYjQxMy01M2EdFGkdRWHJlQ')/transcripts",
-    "@odata.count": 3,
-    "@odata.nextLink": "https://graph.microsoft.com/beta/users('ba321e0d-79ee-478d-8e28-85a19507f456')/onlineMeetings('MSo1N2Y5ZGFjYy03MWJmLTQ3NDMtYjQxMy01M2EdFGkdRWHJlQ')/transcripts?$skiptoken=MSMjMCMjMjAyMS0wOS0xNlQxMzo1OToyNy4xMjEwMzgzWg%3d%3d",
```
[the remark this answers is not in the document]
Comment (v-sdhakshina) [reply api-reference/beta/api/onlinemeeting-list-transcripts.md:128]: Updated as suggested.

**Review by mankadnandan — APPROVED** [review]

**Review by dodaromike — APPROVED** [review]

**Review by dodaromike — APPROVED** [review]

File: api-reference/beta/api/calltranscript-get.md [hunk 2]
```diff
@@ -70,9 +70,6 @@ If successful, this method returns a `200 OK` response code and a [callTranscrip
 
 ## Examples
 
-> [!NOTE]
-> The docx format for transcripts will be deprecated starting May 31, 2023.
```
Comment (Lauragra) [inline api-reference/beta/api/calltranscript-get.md:74]: Why are we deleting the deprecation note when the format was deprecated in May 2023? Is the format deprecated? If so, we need to communicate that to customers. Please undo the deletion of this text and revise the note as follows:

> [!NOTE]
> The docx format for transcripts is deprecated as of May 31, 2023.

Do you know the date on which the format will be removed? See also the guidance around deprecation: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211775/Deprecate-docs

Comment (v-sdhakshina) [reply api-reference/beta/api/calltranscript-get.md:74]: Added a Note as suggested.

File: api-reference/beta/api/onlinemeeting-list-transcripts.md [hunk 3]
```diff
@@ -42,17 +42,6 @@ GET /me/onlineMeetings({meetingId})/transcripts
 GET /users({userId})/onlineMeetings({meetingId})/transcripts
 ```
 
-## Optional query parameters
```
Comment (Lauragra) [inline api-reference/beta/api/onlinemeeting-list-transcripts.md:45]: This H2 section is required in GET method topics. Why are you deleting it? If you need to change the content, please leave the H2 heading. For example, you can say "This method does not support and OData query parameters", if that is the case.

Comment (v-sdhakshina) [reply api-reference/beta/api/onlinemeeting-list-transcripts.md:45]: Retained this section and added Note as suggested.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**